source/docs/03-metadata.xml

```diff
@@ -1441,9 +1441,82 @@
                      </p>
                      <!-- following paragraphs:  
                      <p>Extent and Dimension</p>
-                     <p>Watermark</p>
-                     <p>Stamp</p>
-                     <p>physMedium</p>-->
```

Comment:
@riedde I think you deleted the closing part of the HTML comment tag with your latest commit. Now everything that follows is commented out.